apps/ledger-live-desktop/src/renderer/families/evm/SendAmountFields/PriorityFeeField.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useCallback, useMemo } from "react";
+import invariant from "invariant";
+import BigNumber from "bignumber.js";
+import styled from "styled-components";
+import { useTranslation } from "react-i18next";
+import { Transaction, TransactionStatus } from "@ledgerhq/coin-evm/types";
+import { inferDynamicRange } from "@ledgerhq/live-common/range";
+import { getMainAccount } from "@ledgerhq/live-common/account/index";
+import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
+import { Result } from "@ledgerhq/live-common/bridge/useBridgeTransaction";
+import { Account, AccountBridge, AccountLike } from "@ledgerhq/types-live";
+import { formatCurrencyUnit } from "@ledgerhq/live-common/currencies/index";
+import LabelWithExternalIcon from "~/renderer/components/LabelWithExternalIcon";
+import TranslatedError from "~/renderer/components/TranslatedError";
+import InputCurrency from "~/renderer/components/InputCurrency";
+import { track } from "~/renderer/analytics/segment";
+import Label from "~/renderer/components/Label";
+import { openURL } from "~/renderer/linking";
+import Box from "~/renderer/components/Box";
+import { urls } from "~/config/urls";
+
+const ErrorContainer = styled(Box)<{ hasError?: boolean }>`
+  margin-top: 0px;
+  font-size: 10px;
+  width: 100%;
+  transition: all 0.4s ease-in-out;
+  will-change: max-height;
+  max-height: ${p => (p.hasError ? 60 : 0)}px;
+  min-height: ${p => (p.hasError ? 22 : 0)}px;
+`;
+
+const ErrorDisplay = styled(Box)`
+  color: ${p => p.theme.colors.pearl};
+`;
+
+const WarningDisplay = styled(Box)`
+  color: ${p => p.theme.colors.warning};
+`;
+
+const FeesValues = styled.span`
+  color: ${p => p.theme.colors.neutral.c90};
+`;
+
+const WhiteSpacedLabel = styled(Label)`
+  white-space: pre;
+  color: ${p => p.theme.colors.neutral.c60};
+`;
+
+type Props = {
+  account: AccountLike;
+  parentAccount: Account | null | undefined;
+  transaction: Transaction;
+  status: TransactionStatus;
+  updateTransaction: Result<Transaction>["updateTransaction"];
+};
+
+const fallbackMaxPriorityFeePerGas = inferDynamicRange(new BigNumber(10e9));
```

Comment:
Same thing here, this value is a bit random and I would really like if we could avoid having it without proper justfication of why it's here, and why this 1 Gwei value

Reply:
Same thing here, this value is a bit random and I would really like if we could avoid having it without proper justfication of why it's here, and why this 10 Gwei value.
With the `gasOptions` being mandatory to see this component, I'm not sure we need any fallback

Reply:
Indeed, with the use of `gasOptions` and since it should always be defined at this stage, there is no need for an hardcoded fallback value anymore.